en/cloud_development_setup/instructions.md

```diff
@@ -0,0 +1,24 @@
+You can [skip right over this section](http://tutorial.djangogirls.org/en/installation/#install-python) 
+if you're not using RunCode Cloud Development environment. If you are, your installation experience will be a little different. 
+You can ignore the rest of the installation instructions as you do not need to install anything locally, 
+you just need to create three online accounts. 
+
+## About RunCode
+RunCode is a cloud development environment which people can use without the need to install Python, Django and Visual 
+Studio Code editor locally on their machines. This cloud environment enables anyone to work from any device that has
+an internet connection, including cellphones, tablets, laptop or desktop.
+
+This removes the need for you to install packages on a borrowed laptop or the need for you to own a laptop to attend the workshop or follow this tutorial from home. This will also save you time required for setting up your development
+environment and you can always do the installation later own as a follow-up after finishing the tutorial.
```

Comment:
Full stop after environment. New line - You can always give the installation steps (inc link) a go later as follow-up learning.